File: src/main/java/com/java/learnspark/dataanalysis/chapter02/Course41InitSparkContext.java
package com.java.learnspark.dataanalysis.chapter02;

import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaSparkContext;

public class Course41InitSparkContext {

    public static void main(String[] args) {
        SparkConf conf = new SparkConf().setMaster("local").setAppName("My App");
        JavaSparkContext sc = new JavaSparkContext(conf);
        System.out.println("java init:" + sc);
        sc.close();
    }
}

File: src/main/scala/com/scala/learnspark/dataanalysis/chapter02/Course41InitSparkContext.scala
package com.scala.learnspark.dataanalysis.chapter02

import org.apache.spark.SparkConf
import org.apache.spark.SparkContext

object Course41InitSparkContext {

  def main(args: Array[String]): Unit = {
    val conf = new SparkConf().setMaster("local").setAppName("My App")
    val sc = new SparkContext(conf)
    println("scala init:" + sc)
  }
}
